### PR TITLE
Support eaf-browser-font-family and eaf-terminal-font-family

### PR DESCRIPTION
**Pull request by mackong**
Support eaf-browser-font-family and eaf-terminal-font-family, ex.
```
(eaf-setq eaf-browser-font-family "DejaVu Sans")
(eaf-setq eaf-terminal-font-family "Fira Code")
```

### CODE REVIEW

**Comment by manateelazycat** [comment]
First, thanks for patch.

I think this PR need adjust:
1. Use "" is not empty string as if condition
2. Please remove merge commit

Thanks.

**Comment by mackong** [comment]
1. Sorry, which *if* do you mean, js or python? I have tested, both is ok.
2. merge commit is removed.

File: app/terminal/index.html [hunk 1]
```diff
@@ -44,6 +44,9 @@
           cursorBlink: true,
           theme: theme
       });
+      if ("%6") {
```
Comment (MatthewZMD) [inline app/terminal/index.html:47]: Use "" is not empty string

**Review by MatthewZMD — CHANGES_REQUESTED** [review]

File: app/terminal/index.html (hunk 1 above)
Comment (mackong) [reply app/terminal/index.html:47]: like ```(eaf-setq eaf-terminal-font-family "")```?

Comment (MatthewZMD) [reply app/terminal/index.html:47]: yes

Comment (mackong) [reply app/terminal/index.html:47]: image

After ```(eaf-setq eaf-terminal-font-family "")```, the rendered index.html is like this, and ```if ("")``` is no problem, so what i miss?

Comment (manateelazycat) [reply app/terminal/index.html:47]: I see, patch is merged, thanks for PR ;)

**Pull request merged**